e2e-tests/cypress/tests/support/ui/sidebar_left.ts

```diff
@@ -89,7 +89,7 @@ Cypress.Commands.add('uiGetLhsSection', (section) => {
 });
 
 Cypress.Commands.add('uiBrowseOrCreateChannel', (item) => {
-    cy.findByRole('button', {name: 'Add Channel Dropdown'}).
+    cy.get('.AddChannelDropdown_dropdownButton').
```

Comment:
I haven't tried but could this be as below?
```suggestion
    cy.uiGetLHSAddChannelButton().
```